### PR TITLE
version bump

### PR DESCRIPTION
**Pull request by omerlh**
see Soluto/kamus#326

### CODE REVIEW

File: charts/kamus/README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ The chart can be customized using the following configurable parameters. Most se
 | `keyManagement.awsKms.region`                | AWS KMS region   | 
 | `keyManagement.awsKms.cmkPrefix`                | Prefix for the customer master keys that are created in the KMS  | 
 | `pod.annotations`                             | Set the annotations for Kamus's pods | 
-| `crdConersationEnabled`                       | See bellow, relevant if running Kamus on a cluster version bellow 1.15.0 |
+| `crdConersationEnabled`                       | See bellow, relevant if running Kamus on a cluster version bellow |
```
Comment (shaikatz) [inline charts/kamus/README.md]: Why did you remove the 1.15.0?

Comment (omerlh) [reply charts/kamus/README.md]: by mistake, fixed

**Comment by omerlh** [comment]
fixed @shaikatz
